modules/minimp3/audio_stream_mp3.cpp

```diff
@@ -46,7 +46,9 @@ int AudioStreamPlaybackMP3::_mix_internal(AudioFrame *p_buffer, int p_frames) {
 	int frames_mixed_this_step = p_frames;
 
 	int beat_length_frames = -1;
-	bool beat_loop = mp3_stream->has_loop() && mp3_stream->get_bpm() > 0 && mp3_stream->get_beat_count() > 0;
+	bool use_loop = looping_override ? looping : mp3_stream->loop;
```

Comment:
I don't know about this. Maybe the parameter for `looping` shouldn't be a binary, but rather an enum: DEFAULT, ON, OFF

In my mind, we should be able to let the looping parameter to DEFAULT and loop the sample if the stream itself is set to looping.

Reply:
The property itself is checkable, like theme overrides. So you can choose whether use the default or a custom value.

Reply:
![image](https://github.com/godotengine/godot/assets/270928/a8d0bb1a-4763-43a1-92ce-6169229cc231)

Just built the PR to make sure and you're right.

Reply:
Right, the original screenshot is outdated.